packages/stateManager/index.ts

```diff
@@ -330,6 +329,29 @@ class ShopManifestManager extends PublicObjectManager<ShopManifest> {
         }
         manifest.addAcceptedCurrencies = filtered;
       }
+      if (um.addPayee) {
```

Comment:
* **note to self** do we have removePayee in the vectors?

Reply:
we don't. we only have `remove_accepted_currencies` right now.

Reply:
but let's add crud tests for

* removing a payee that doesn't exist (should throw an _not found_ error)
* add a payee with the same name (should throw an _already exists_ error)

the relay would _also_ tell you this but i think it would be nicer to catch this before sending a request over the network.